include/openssl/hpke.h

```diff
@@ -147,5 +147,15 @@ int OSSL_HPKE_str2suite(const char *str, OSSL_HPKE_SUITE *suite);
 size_t OSSL_HPKE_get_ciphertext_size(OSSL_HPKE_SUITE suite, size_t clearlen);
 size_t OSSL_HPKE_get_public_encap_size(OSSL_HPKE_SUITE suite);
 size_t OSSL_HPKE_get_recommended_ikmelen(OSSL_HPKE_SUITE suite);
+int OSSL_HPKE_BOX_seal(OSSL_LIB_CTX *libctx, const char *propq,
```

Comment:
What does BOX actually mean? (What another library does is not part of the answer :))

Reply:
The reason for BOX is that this is the best analogy for an integrated AEAD and encapsulation operation. I'm open to alternative names.

Reply:
I think if you look at RFC9180 what you have is just the API in 6.1 (I would add the aad as an optional param)..
which is the single shot API?